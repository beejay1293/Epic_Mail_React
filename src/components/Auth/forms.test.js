import React from 'react';
import { mount, shallow, render } from 'enzyme';
import Forms from './forms';
describe('Form', () => {
  it('should render correctly the register form', () => {
    const Form = shallow(<Forms />);
  
    expect(Form).toMatchSnapshot();
  });
});