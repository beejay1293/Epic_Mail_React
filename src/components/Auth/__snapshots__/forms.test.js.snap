// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form should render correctly the register form 1`] = `
<div
  className="login"
>
  <h3
    className="sub_header"
  />
  <h3
    className="sub_header"
  >
    Create an EPIC Mail Account
  </h3>
  <Connect(withRouter(Register)) />
  <div
    className="signup"
  >
    <p>
      Already have an account?
      <em
        className="signinbtn actionbtn"
        onClick={[Function]}
      >
        Sign in.
      </em>
    </p>
  </div>
</div>
`;
